ztp/kube-compare-reference/Makefile

```diff
@@ -24,4 +27,32 @@ metadata_lint: kubectl-cluster_compare
 
 .PHONY: clean
 clean:
-	rm -rf kubectl-cluster_compare
+	rm -rf kubectl-cluster_compare Chartv1 renderedv1
+
+
+.PHONY: convert
+convert: $(HELMCONVERTOR) helm
+	@if [ -e default_value.yaml ]; then \
+		$(HELMCONVERTOR) -r ./metadata.yaml -n Chartv1 -v default_value.yaml; \
+	else \
+		echo "Error: default_value.yaml doesn't exist!"; \
+		exit 0; \
+	fi
+	@helm template renderedv1 ./Chartv1 --output-dir renderedv1
+
+$(HELMCONVERTOR):
+	go install github.com/openshift/kube-compare/addon-tools/helm-convert@latest
+
+helm:
+	curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash
+
+
+.PHONY: compare
+compare: convert
+	@if [ -e compare.sh ]; then \
+		chmod 700 ./compare.sh; \
+		./compare.sh; \
+	else \
+		echo "Error: compare.sh no found!"; \
```

Comment:
I would not have the if and chmod; just check-in compare.sh with +x permissions, and just run ./compare.sh